lib/cloudregister/registerutils.py

```diff
@@ -567,6 +567,52 @@ def set_registry_auth_token(registry_fqdn, username, password):
     return True
 
 
+# ----------------------------------------------------------------------------
+def clean_registry_setup():
+    """Remove the data previously set to make the registry work."""
+    remove_auth_token()
+
+
+# ----------------------------------------------------------------------------
+def remove_auth_token():
+    """Remove the auth token from the config json file."""
+    if not os.path.exists(REGISTRY_CREDENTIALS_PATH):
+        return
+
+    smt = get_smt_from_store(__get_registered_smt_file_path())
+    if not smt:
+        return
+
+    logging.info('Unsetting the auth entry for %s' % smt.get_registry_FQDN())
+    try:
+        with open(REGISTRY_CREDENTIALS_PATH, 'r') as cred_json:
+            config_json = json.load(cred_json)
+        config_auths = config_json['auths']
+        # unset the registry credentials
+        del config_auths[smt.get_registry_FQDN()]
+    except KeyError:
+        logging.info('No auth key present. Nothing to do.')
+        return True
+    except json.decoder.JSONDecodeError:
+        logging.info(
+            'Could not unset the registry credentials: '
+            'Error found when opening %s' % REGISTRY_CREDENTIALS_PATH
+        )
+        return
+
+    try:
```

Comment:
this `try/except` block of updating the `/etc/containers/config.json` file can has its own method so it can be reused in `set_registry_auth_token()` as well

I plan to do that once the registry PRs are merged

Reply:
Created the method in #159, no reason to wait